packages/server/lib/browsers/electron.js

```diff
@@ -198,8 +196,6 @@ module.exports = {
     }
 
     return this._launch(win, url, automation, preferences)
-    .tap(_maybeRecordVideo(win.webContents, preferences))
```

Comment:
Good to see less async callbacks using `tap`. I'm sure it's an antipattern to use `tap` for anything other than logging, as noted by the first comment on the `tap` docs: http://bluebirdjs.com/docs/api/tap.html

I am pretty sure `tap` will never wait, and previously `maybeRecordVideo` was an async function, so this was basically a race condition waiting to happen.